fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -105,4 +105,16 @@ public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constra
     protected Map<String, List<Interval>> getLatestAvailableIntervalsByTable() {
         return metadataService.getAvailableIntervalsByTable(name);
     }
+
+    public DataSourceMetadataService getMetadataService() {
+        return metadataService;
+    }
+
+    public TableName getName() {
+        return name;
+    }
+
+    public Set<String> getCachedColumnNames() {
+        return cachedColumnNames;
+    }
```

Comment:
Let's drop the 'cached' prefix on the member variables in this class and in the getter.